modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -23,10 +23,10 @@ The minimum values for the amount of CPU and memory allocated to each replica of
 
 Anypoint Platform allocates native and heap memory for a deployed application. Heap memory is the portion of the total memory that is made available to the Mule runtime and the application. Heap memory is used for tasks like handling payload.
 
-Both CloudHub and Runtime fabric allocate both types of memory. However, they differ in how the memory allocation of each memory type is calculated.
+Both CloudHub and Runtime Fabric allocate both types of memory. However, they differ in how the memory allocation of each memory type is calculated.
```

Comment:
Per previous comment, I would spell out "Anypoint Runtime Fabric" here. I'm not advocating tiresome repetition, but our product naming has been so inconsistently implemented that I think some repairs are in order. Remind me to tell you about all of my many discussions with both Salesforce and MuleSoft legal reps....